engines/director/detection_tables.h

```diff
@@ -2172,7 +2172,7 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME1("adamtis", "v2.01",	  "ADAMPLAY.EXE",	   "803570c5edef30938a1efb0900f113e0", 695855, 404),
 	WINDEMO1("adamtis", "Skull Demo", "PUZZLE.EXE",		   "d2dcc8b55382fc1807e357af5108f7ba", 1057997, 404),
 
-	MACGAME1("alice", "", "Alice", "3b61149c922f0fd815ca29686e4f554a", 304714, 400),
+	MACGAME1("alice", "", "Alice", "3b61149c922f0fd815ca29686e4f554a", 304458, 400),
```

Comment:
Can you confirm that this is the Digipak release? If so, then we can mark it as such.

Reply:
This is the English-language rerelease. The Digipak release is the "Rerelease" one above that's using D2.